### ISSUE TITLE
No alternative text for options within viewer 

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#1537]
![screen shot 2017-01-24 at 15 24 58](https://cloud.githubusercontent.com/assets/13200338/22253394/5bdc8fe2-e249-11e6-9804-01c3a5aaf4e5.png)
![screen shot 2017-01-24 at 15 24 55](https://cloud.githubusercontent.com/assets/13200338/22253395/5be2c038-e249-11e6-89d2-7dc636118d75.png)
